extensions/notebook/package.json

```diff
@@ -19,6 +19,9 @@
       "supported": true
     }
   },
+  "enabledApiProposals": [
```

Comment:
Do you still need this with the typings import in refs.d.ts?

Reply:
Yeah, this time it was throwing a build error without it.